session/session.go

```diff
@@ -1086,6 +1087,21 @@ func BootstrapSession(store kv.Storage) (*domain.Domain, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+
+	// get system tz from mysql.tidb
+	callback := func() string {
+		sql := `select variable_value from mysql.tidb where variable_name = "system_tz"`
+		rss, errLoad := se.Execute(context.Background(), sql)
```

Comment:
`se` may be used in another goroutine started by `dom.LoadPrivilegeLoop`, it is not thread-safe.